RAGchain/preprocess/loader/hwp_table_loader.py

```diff
@@ -0,0 +1,152 @@
+import os
+import re
+import zipfile
+import shutil
+from typing import List
+
+import win32com.client as win32
```

Comment:
make try-except when importing win32com
Because mac OS and linux can't run this package.